Resources/Textures/_Umbra/Objects/Weapons/Guns/Pistols/sandstone_taipan.rsi/meta.json

```
@@ -0,0 +1,39 @@
+{
+    "version": 1,
+    "license": "CC-BY-SA-3.0",
+    "copyright": "Sprited by akirrer for Sector Umbra",
```

```suggestion
    "copyright": "Sprited by akirrer (Github) for Sector Umbra",
```
Also, does this have a source/reference image? If so it should be included here.